javascript/ql/experimental/adaptivethreatmodeling/modelbuilding/extraction/ExtractEndpointMapping.ql

```diff
@@ -8,21 +8,41 @@ import experimental.adaptivethreatmodeling.SqlInjectionATM as SqlInjectionATM
 import experimental.adaptivethreatmodeling.NosqlInjectionATM as NosqlInjectionATM
 import experimental.adaptivethreatmodeling.TaintedPathATM as TaintedPathATM
 import experimental.adaptivethreatmodeling.XssATM as XssATM
+import experimental.adaptivethreatmodeling.StoredXssATM as StoredXssATM
+import experimental.adaptivethreatmodeling.XssThroughDomATM as XssThroughDomATM
 import experimental.adaptivethreatmodeling.AdaptiveThreatModeling
 
-from string queryName, AtmConfig c, EndpointType e
+from string queryName, ATMConfig c, int endpointTypeEncoded
 where
   (
-    queryName = "SqlInjectionATM.ql" and
-    c instanceof SqlInjectionATM::SqlInjectionAtmConfig
+    queryName = "Unknown" and
+    endpointTypeEncoded = 0
     or
-    queryName = "NosqlInjectionATM.ql" and
-    c instanceof NosqlInjectionATM::NosqlInjectionAtmConfig
+    queryName = "NotASink" and
+    endpointTypeEncoded = 0
```

Comment:
We should remove these two disjuncts while the proposal about these is still in flux.